modules/@apostrophecms/doc-type/index.js

```diff
@@ -1067,6 +1060,14 @@ module.exports = {
         const existing = await actionModule.findForEditing(toReq, {
           _id: toId
         }).permission('view').toObject();
+
+        const eventOptions = {
+          source: draft.aposLocale.split(':')[0],
+          target: toLocale,
+          docType: draft.type,
+          existing: Boolean(existing)
```

Comment:
This replaces `update` and says if we want to translate or not (if already exists in the other locale we don't want to translate).